ledger-tool/src/main.rs

```diff
@@ -1396,7 +1396,11 @@ fn main() {
             .hidden(true);
     let ancient_append_vecs = Arg::with_name("accounts_db_ancient_append_vecs")
         .long("accounts-db-ancient-append-vecs")
-        .help("AppendVecs that are older than an epoch are squashed together.")
+        .value_name("SLOT-OFFSET")
+        .validator(is_parsable::<u64>)
+        .takes_value(true)
+        .default_value("0")
+        .help("AppendVecs that are older than an epoch (- SLOT-OFFSET) are squashed together.")
```

Comment:
To me, the subtraction in `an epoch (- SLOT-OFFSET)` is confusing. It reads as the offset pushing the squash further back. So if we had offset of 1 and epoch of 432000 slots then we'd squash if slot was older than `current_slot - 432001`.

Thoughts on `older than (slots_per_epoch - SLOT-OFFSET)`?